pex/pex.py

```diff
@@ -444,7 +444,7 @@ def execute_pkg_resources(spec):
     else:
       # setuptools < 11.3
       runner = entry.load(require=False)
-    runner()
+    sys.exit(runner())
```

Comment:
I thought about that, but most packages will already have done a `sys.exit()` or `raise SystemExit` here anyways, [including pex](https://github.com/pantsbuild/pex/blob/v1.1.10/pex/bin/pex.py#L557). I can still move it if you'd like